java/clients/src/main/java/sleeper/clients/teardown/RemoveECRRepositories.java

```diff
@@ -53,8 +49,8 @@ public static void remove(EcrClient ecr, InstanceProperties properties, List<Str
     }
 
     private static Stream<String> repositoryNamesFromProperties(InstanceProperties properties) {
-        return Stream.of(ECR_COMPACTION_REPO, ECR_INGEST_REPO,
-                BULK_EXPORT_ECR_REPO, BULK_IMPORT_REPO, BULK_IMPORT_EMR_SERVERLESS_CUSTOM_IMAGE_REPO)
+        return Stream.of(ECR_REPOSITORY_PREFIX,
+                BULK_EXPORT_ECR_REPO)
```

Comment:
I'm not sure if this is the right behaviour for this method. Currently this method returns the names of ECR repositories that are set for the instance. Usually that will just be all the repositories. Previously, it was possible that some of those repositories could be not set at all in the instance properties, and in that case it wouldn't use them.

This is used to delete the repositories by their name. It probably makes sense to still look for all of the repositories.

I was wondering if we should model whether we expect the repositories to exist for the instance, but we probably still want to check if the repositories exist anyway, and we also want to delete this code completely soon and move the behaviour into the CDK.

Reply:
I think this needs changing to compute all the repository names.